mesonbuild/interpreter/interpreter.py

```diff
@@ -3203,13 +3203,13 @@ def build_target_decorator_caller(self, node, args, kwargs):
         if 'sources' in kwargs:
             sources += listify(kwargs['sources'])
         if any(isinstance(s, build.BuildTarget) for s in sources):
-            FeatureDeprecated.single_use('passing references to built targets as a source file', '1.1.0', self.subproject,
-                                         'consider using `link_with` or `link_whole` if you meant to link, or dropping them as otherwise they are ignored',
-                                         node)
+            FeatureBroken.single_use('passing references to built targets as a source file', '1.1.0', self.subproject,
+                                     'consider using `link_with` or `link_whole` if you meant to link, or dropping them as otherwise they are ignored',
```

Comment:
You capitalized your description for the int/bool issue, but these are lowercased. We should probably converge on the same style.